release_notes/ocp-4-20-release-notes.adoc

```diff
@@ -794,6 +794,15 @@ For more information, see xref:../networking/hardware_networks/configure-lacp-fo
 ==== Network policies for additional namespaces
 With this release, {product-title} deploys Kubernetes network policies to additional system namespaces to control ingress and egress traffic. It is anticipated that future releases might include network policies for additional system namespaces and Red{nbsp}Hat Operators.
 
+[id="ocp-4-20-ptp-holdover_{context}"]
+==== Unassisted holderover for PTP devices (Technology Preview)
+With this release, the PTP Operator provides unassisted holdover as a Technology Preview feature. When the upstream timing signal is lost, the PTP Operator automatically places PTP devices configured as either a boundary clock or a time slave clock into holdover mode. This helps maintain a continuous and stable time source for cluster nodes, minimizing time synchronization disruptions.
```

Comment:
Automatic placement into holdover mode maintains a continuous and stable time source for cluster nodes, minimizing time synchronization disruptions.